adapters/adform/adform.go

```diff
@@ -354,6 +373,7 @@ func toOpenRtbBids(adformBids []*adformBid, r *openrtb.BidRequest) []*adapters.T
 			W:      bid.Width,
 			H:      bid.Height,
 			DealID: bid.DealId,
+			CrID:   bid.CreativeId,
```

Comment:
func `toPBSBidSlice` should have the same assignment `Creative_id: bid.CreativeId,`

Reply:
added, thank you